### PR TITLE
chore: integrate split button

### LINKED ISSUE #174: Create split button and add it to profile page
## Description

Create the split button, using `dropdown` component, and add it to the profile page

## Acceptance Criteria

- [x] Implement split button according to the designs
- [x] Use it in the profile page

## Pull Requests

[`<SplitButton>`](https://github.com/ipfs-shipyard/nomios-web-uikit/pull/73)

[Split button integration](https://github.com/ipfs-shipyard/nomios-web/pull/28)

### Related PRs

- [Add remove, actions and trash icons](https://github.com/ipfs-shipyard/nomios-web-uikit/pull/72)

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
This PR adds the split button to the identity page.

Closes [#174](https://github.com/ipfs-shipyard/pm-idm/issues/174).

### CODE REVIEW

File: src/pages/identity/IdentityDetails.js [hunk 1]
```diff
@@ -43,28 +55,63 @@ class IdentityDetails extends Component {
...
+                        actions={ actions }
+                        mainActionId={ mainActionId }
+                        onActionClick={ this.handleActionClick } />
                 </div>
             </div>
         );
     }
 
-    handleEdit = () => {
-        alert('Edit profile');
```
Comment (satazor) [inline src/pages/identity/IdentityDetails.js:57]: Can’t we use memoizeOne directly here?

**Comment by satazor** [comment]
Just a minor comment, looks good!

File: src/pages/identity/IdentityDetails.js (hunk 1 above)
Comment (PedroMiguelSS) [reply src/pages/identity/IdentityDetails.js:57]: Yes, we can, but then eslint will ask us to place this code before the constructor. What should we do?

Comment (satazor) [reply src/pages/identity/IdentityDetails.js:57]: Damn, we really need to fix that eslint rule. Is the order rule right?

Comment (satazor) [reply src/pages/identity/IdentityDetails.js:57]: Perhaps we can drop a eslint ignore for now?

Comment (PedroMiguelSS) [reply src/pages/identity/IdentityDetails.js:57]: Sure! I need to change this kind of stuff in some other components.

**Review by satazor — APPROVED** [review]

**Pull request merged**